images/base/scripts/common/setup.sh

```diff
@@ -83,7 +83,12 @@ fi
 
 # First install protoc
 cd /tmp
-git clone https://github.com/google/protobuf.git
+if [ x$MACHINE = xs390x ]
+then
+    git clone https://github.com/gongsu832/protobuf.git
```

Comment:
Either you clone the original protobuf and apply the patch yourself, or you pre-apply the patch and clone the patched version. I don't see much difference between the two. Either way, someone has to maintain the patch.